### ISSUE TITLE
[BUG] forbidden to create namespace "__namespace"

### ISSUE THREAD

**Issue by mathspanda** [apache/kvrocks#770]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-kvrocks/issues) and found no similar issues.

### Version

341a989d75783d8dad76bbd5a4473dd345472337

### Minimal reproduce step

```
./redis-cli -p 6666
127.0.0.1:6666> auth XXX
OK
127.0.0.1:6666> namespace add ns1 token1
OK
127.0.0.1:6666> namespace add ns2 token2
OK
127.0.0.1:6666> namespace get *
1) "ns1"
2) "token1"
3) "ns2"
4) "token2"
127.0.0.1:6666> namespace add __namespace token3
OK
127.0.0.1:6666> namespace get *
1) "ns1"
2) "token1"
3) "ns2"
4) "token2"
5) "__namespace"
6) "token3"
```

### What did you expect to see?

1. Forbidden to create namespace "__namespace"
2. Add default namespace "__namespace" into the results of "namespace get *"

### What did you see instead?

Pls See `Minimal reproduce step`

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

**Comment by git-hulk** [comment]
@mathspanda good catch

**Comment by git-hulk** [comment]
closed by #772
